components/oauth/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth2/dao/TokenMgtDAO.java

```diff
@@ -71,7 +79,8 @@
         try {
             maxPoolSize =
                     Integer.parseInt(IdentityUtil.getProperty("JDBCPersistenceManager.SessionDataPersist.PoolSize"));
-        } catch (Exception e) {
+        } catch (NumberFormatException e) {
+            log.error("Error while parsing the integer", e);
```

Comment:
Better to make this message more meaningful with following information
* what was the name of property
* what was the value given
* what was the effect (in this case it's being assigned to a default value)